Tools/TestWebKitAPI/glib/CMakeLists.txt

```
@@ -143,6 +144,25 @@ if (ENABLE_WPE_PLATFORM)
     if (COMPILER_IS_GCC_OR_CLANG)
         WEBKIT_ADD_TARGET_CXX_FLAGS(TestWPESettings -Wno-unused-parameter)
     endif ()
+
+    add_executable(TestWPEDisplay
+        ${TOOLS_DIR}/TestWebKitAPI/glib/WebKitGLib/TestMain.cpp
+        ${TOOLS_DIR}/TestWebKitAPI/Tests/WPEPlatform/TestDisplay.cpp
+    )
+    target_link_libraries(TestWPEDisplay
+        ${WebKitGLibAPITest_LIBRARIES}
+        WPEPlatform-${WPE_API_VERSION}
+    )
+    target_include_directories(TestWPEDisplay PRIVATE
+        ${WebKitGLibAPITests_INCLUDE_DIRECTORIES}
+        ${WPEPlatform_DERIVED_SOURCES_DIR}
+        ${WEBKIT_DIR}/WPEPlatform
+    )
+    target_compile_definitions(TestWPEDisplay PRIVATE ${WebKitGLibAPITests_DEFINITIONS})
+    set_target_properties(TestWPEDisplay PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/TestWebKitAPI/WPEPlatform)
+    if (COMPILER_IS_GCC_OR_CLANG)
+        WEBKIT_ADD_TARGET_CXX_FLAGS(TestWPEDisplay -Wno-unused-parameter)
+    endif ()
```

One may use `WEBKIT_ADD_TARGET_CXX_FLAGS()` without checking `COMPILER_IS_GC_OR_CLANG` beforehand. The function actually checks whether the compiler supports the given flags and will only add them if supported.

```suggestion
    WEBKIT_ADD_TARGET_CXX_FLAGS(TestWPEDisplay -Wno-unused-parameter)
```

---

Done!